src/index.js

```diff
@@ -91,7 +91,9 @@ export { default as PasswordInput } from './components/inputs/password-input';
 export { default as SelectInput } from './components/inputs/select-input';
 export { default as TextInput } from './components/inputs/text-input';
 export { default as TimeInput } from './components/inputs/time-input';
-
+export { default as Toggle } from './components/inputs/toggle';
+export { default as Checkbox } from './components/inputs/checkbox';
+export { default as Radio } from './components/inputs/radio';
```

Comment:
I think we should name them consistently with the other component, so with `Input` suffix.

Reply:
Very valid!

Reply:
I renamed `Toggle` -> `ToggleInput`. If we are happy with this, I will rename the rest of them on Monday